socorro/processor/rules/breakpad.py

```diff
@@ -397,68 +376,72 @@ def run_stackwalker(self, crash_id, command_path, command_line, status):
-            for dump_name, dump_file_path in dumps.items():
-                # This rule only works on minidumps which the crash reporter prefixes
-                # with the value of dump_field (defaults to "upload_file_minidump")
-                if not dump_name.startswith(self.dump_field):
-                    continue
-
-                file_size = os.path.getsize(dump_file_path)
-                if file_size == 0:
-                    # If the dump file is empty (0-bytes), then we don't want to bother
-                    # running minidump-stackwalker.
-                    #
-                    # This is a bad case, so we want to add a note. However, since this
-                    # is a shortcut, we also include some stackwalker_data.
-                    stackwalker_data = {
-                        "mdsw_status_string": "EmptyMinidump",
-                        "mdsw_stderr": "Shortcut for 0-bytes minidump.",
-                    }
-
-                    status.add_note(
-                        f"MinidumpStackwalkRule: {dump_name} is empty--skipping "
-                        + "minidump processing"
-                    )
-
-                else:
-                    command_line = self.expand_commandline(
-                        dump_file_path=dump_file_path,
-                        raw_crash_path=raw_crash_path,
-                    )
-
-                    stackwalker_data = self.run_stackwalker(
-                        crash_id=crash_id,
-                        command_path=self.command_path,
-                        command_line=command_line,
-                        status=status,
-                    )
-
-                    stderr = stackwalker_data.get("mdsw_stderr", "").strip()
-                    if stderr:
-                        if stderr.startswith("ERROR"):
-                            indicator = stderr.split(" ")[1]
-                        else:
-                            indicator = ""
-
-                        status_string = stackwalker_data.get("mdsw_status_string", "")
-                        if indicator and status_string in ["OK", "unknown error"]:
-                            stackwalker_data["mdsw_status_string"] = indicator
-                            status.add_note(
-                                f"MinidumpStackwalkRule: processing {dump_name} had error; "
-                                + "stomped on mdsw_status_string"
-                            )
-
-                if dump_name == self.dump_field:
-                    processed_crash.update(stackwalker_data)
-
-                else:
-                    if dump_name not in processed_crash["additional_minidumps"]:
-                        processed_crash["additional_minidumps"].append(dump_name)
-                    processed_crash.setdefault(dump_name, {})
-                    processed_crash[dump_name].update(stackwalker_data)
+        # Save crash annotations to disk for stackwalker to look at
```

Comment:
This is mostly the same except it's dedented because we no longer have the `tmp_raw_crash_file` context.